sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/TableRowToStorageApiProtoTest.java

```diff
@@ -576,8 +576,8 @@ public void testNestedFromTableSchema() {
                   new TableCell().setV("00:52:07.123456"),
                   new TableCell().setV("2019-08-16T00:52:07.123456"),
                   new TableCell().setV("2019-08-16"),
-                  new TableCell().setV("23.4"),
```

Comment:
why does this get changed, any breaking change?

Reply:
I simply updated the test to pass in an integer rather than string. This helps to demonstrate the problem, and exercises the fix.

Reply:
I see, I think we should add a test case instead of replace one. Now the conversion of double to decimal becomes untested

Reply:
I have restored the original test case and added new ones to exercise integers.

Reply:
Thanks!